src/helpers/useUndo.ts

```diff
@@ -80,7 +80,7 @@ export interface UndoResult {
   /**
    * Clear state.
    */
-  clear: () => void;
+  clear: (nodes: NodeData[], edges: EdgeData[]) => void;
```

Comment:
I guess this is technically a breaking api change?

Reply:
now clear() not working at all, just clean a whole tree.
so technically yes, but in practice, nobody could use it